src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return a => {
+    return b => {
```

Comment:
You can add 0 as a default value. So if no argument is passed, it would still work.